README.md

```diff
@@ -48,4 +48,4 @@ Extend @ni/eslint-config/typescript in the ESLint configuration. Configure the @
 
 ## License
 
-[MIT (c) 2020 National Instruments Corporation](./LICENSE)
\ No newline at end of file
+[MIT (c) 2020 National Instruments Corporation](./LICENSE)
```

Comment:
Should we update this to 2021?

Reply:
Good suggestion. I ended up doing this in #21